includes/database/queries/class-order.php

```diff
@@ -164,19 +164,161 @@ public function __construct( $query = array() ) {
 
 		// In EDD 3.0 we converted our use of the status 'publish' to 'complete', this accounts for queries using publish.
 		if ( isset( $query['status'] ) ) {
-			if ( is_array( $query['status'] ) && in_array( 'publish',  $query['status'] ) ) {
+			if ( is_array( $query['status'] ) && in_array( 'publish', $query['status'], true ) ) {
 				foreach ( $query['status'] as $key => $status ) {
 					if ( 'publish' === $status ) {
 						unset( $query['status'][ $key ] );
 					}
 				}
 
 				$query['status'][] = 'complete';
-			} else if ( 'publish' === $query['status'] ) {
+			} elseif ( 'publish' === $query['status'] ) {
 				$query['status'] = 'complete';
 			}
 		}
 
 		parent::__construct( $query );
 	}
+
+	/**
+	 * Set up the filter callback to add the country and region from the order addresses table.
+	 *
+	 * @since 3.0
+	 * @access public
+	 *
+	 * @param string|array $query {
```

Comment:
Maybe we should add `country` and `region` to these PHPDocs and `__construct()`.